src/test/java/com/cloudbees/jenkins/support/impl/RunningJobsTest.java

```diff
@@ -0,0 +1,30 @@
+package com.cloudbees.jenkins.support.impl;
+
+import com.cloudbees.jenkins.support.SupportTestUtils;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.queue.QueueTaskFuture;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertThat;
+
+public class RunningJobsTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void addContents() throws Exception {
+        String jobName = "running-job";
+        FreeStyleProject p = j.createFreeStyleProject(jobName);
```

Comment:
Really need to import the BOM, add Pipeline test dependencies, and verify behavior for a Pipeline build running in a couple of `node` blocks in `parallel` to see what this would look like.

Reply:
@jglick I assume you are referring to https://github.com/jenkinsci/bom

Do you have an example of what you are asking me to do? I'd rather not play PR ping pong trying to guess.

Reply:
imported BOM and added a Pipeline test.

NOTE: Required bumping required from 2.107 -> 2.138.4

Reply:
To be clear, the BOM part is just a convenience, but anyway there is little reason to still be putting out releases for very old LTS lines.